src/components/Calculator.test.js

```diff
@@ -0,0 +1,7 @@
+import renderer from 'react-test-renderer';
+import Calculator from './Calculator';
+
+it('Calculator renders correctly', () => {
+  const tree = renderer.create(<Calculator />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
## [OPTIONAL]

- [ ] kindly note that it's always a best  practice to group all tests in one folder